.gitignore

```diff
@@ -2,4 +2,5 @@ tests/*/results.json
 tests/*/ALLTESTS
 tests/*/ParserErrorLog.txt
 tests/*/test
-tests/*/test.cob
\ No newline at end of file
+tests/*/test.cob
+*~
```

Comment:
This entry does not belong as it has nothing to do with the project.

Also, the last line is missing an `EOL`.